kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Food2Door/attributes/Products.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.good.patterns.challenges.Food2Door.attributes;
+
+public class Products {
+    private final String productName;
+    private final int productQuantity;
+
+    public Products(final String productName, final int productQuantity) {
+        this.productName = productName;
+        this.productQuantity = productQuantity;
+    }
+
+    public String getProductName() {
+        return productName;
+    }
+
+    public int getProductQuantity() {
+        return productQuantity;
+    }
```

Comment:
ten program to tylko symulacja, ale dobrze bylo jakos wykorzystac productName i productQuantity. Np. rzucac wyjatkiem jak quantity jest 0 < albo jak productName.isEmpty()